action.yml

```
@@ -39,6 +39,9 @@ inputs:
   auto-merge:
     description: "Enable auto merge for pull request. Requires auto merging to be enabled in repository settings"
     required: false
+  disable-auto-update-pr:
+    description: "Disable the title and body of existing pull request automatically updated."
+    required: false
```

I think I'd prefer the option to be called `update-pull-request-title-and-body`

```suggestion
  update-pull-request-title-and-body:
    description: "Set to true if you want the title and body to be updated for an existing pull request"
    default: false
    required: false
```

---

Thanks for the suggestion. I'll write the test this weekend.